### ISSUE TITLE
[Feature Request]: Ability to insert rich text into template

### ISSUE THREAD

**Issue by exzizt** [Ideolys/carbone#115]
**Problem to solve**
It would be nice to be able to (or is it already somehow possible?) insert blocks of rich text into a template.

**Proposed solution**
A function, maybe named "convRichText()", to render the passed-in string/chunk of rich text into the document.

**Describe alternatives you've considered**
It might be possible to somehow achieve this outside of Carbone by injecting the rich text into the DOCX template before passing it to Carbone for rendering.

**Additional context**
N/A

**Comment by steevepay** [comment]
Hello @exzizt, the rich text is mainly HTML content?

**Comment by exzizt** [comment]
> Hello @exzizt, the rich text is mainly HTML content?

Yes, I believe the rich text form inputs return "rich text" as HTML. We will likey be using [this](https://ej2.syncfusion.com/react/documentation/rich-text-editor/getting-started/).

**Comment by steevepay** [comment]
The Carbone Enterprise Edition already provides an HTML conversion for DOCX and ODT reports. We don't plan to open source it.
We didn't update the website documentation yet but reach us if you are interested.

**Comment by exzizt** [comment]
> The Carbone Enterprise Edition already provides an HTML conversion for DOCX and ODT reports. We don't plan to open source it.
> We didn't update the website documentation yet but reach us if you are interested.

To clarify, we would like to render richtext from HTML inside of a DOCX template. Is this not possible? Or only possible if using the Enterprise edition?

We may end up rolling our own, similar to what we had to do to achieve dynamic image insertion/replacement.

**Comment by steevepay** [comment]
It is only possible with the Carbone Enterprise Edition.

**Comment by txinto** [comment]
What if rich text means "markdown"?  This means, the JSON have a text in Markdown, and Carbone renders it inside the ODT file.  Thanks.

**Comment by alstarace** [comment]
We're also interested.
Where can we find some documentation on it?
Can we test it before with the free account and/or on the carbone studio?

**Comment by cjk2** [comment]
That is also interesting for me. It would be important to test it in advance. Are there any possibilities?
